docs/.vuepress/theme/utils/translations.js

```diff
@@ -363,6 +363,96 @@ const translations = {
     'page-learn': '배우기',
     'page-developers': '개발자 가이드'
   },
+  'nl-NL': {
```

Comment:
I should have specified this (will do for future translations) but please update this key to match language code used by CrowdIn (in this case `nl` - https://crowdin.com/project/ethereumfoundation/nl). This is important for displaying incomplete languages on ethereum.org/languages (you can see we use these keys to compare [here](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/.vuepress/components/LanguagesPage.vue#L53).